src/ensembl/src/content/app/entity-viewer/shared/rest/rest-adaptors/rest-protein-adaptor.ts

```diff
@@ -0,0 +1,10 @@
+import { ProteinStatsInResponse } from '../rest-data-fetchers/proteinData';
+
+export const restProteinStatsAdaptor = (
+  proteinStats: ProteinStatsInResponse
+) => ({
+  structures: proteinStats.pdbs,
+  ligands: proteinStats.ligands,
+  interactions: proteinStats.interaction_partners,
+  annotations: proteinStats.annotations
```

Comment:
since they are all numbers, maybe make the field names more explicit: (structuresCount, ligandsCount, interactionsCount, annotationsCount)?

Reply:
✔️